src/main/steps/applicant2/you-cannot-apply/content.ts

```diff
@@ -0,0 +1,20 @@
+import { TranslationFn } from '../../../app/controller/GetController';
```

Comment:
Does this need to be added to `src/main/steps/applicant2Sequence.ts`?

Reply:
@velimd has a PR open with it added to the sequence